changelog/1.15.1.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "1.15.1"
+description: "Released on 01/25.2021."
+---
+
+### Breaking Changes ❗
```

Comment:
We should link to the 1.15.0 documentation here instead and then include just the bug fix here